Source/Scene/Model.js

```diff
@@ -2972,11 +3069,58 @@ define([
                     castShadows : castShadows,
                     receiveShadows : receiveShadows,
                     uniformMap : uniformMap,
-                    renderState : rs,
+                    renderState : drawRS,
                     owner : owner,
                     pass : isTranslucent ? Pass.TRANSLUCENT : Pass.OPAQUE
                 });
 
+                // Setup the stencil command for the highlight
+                var highlightRS = clone(rs);
```

Comment:
Shouldn't the blend state for this depend on the alpha value of `hightlightColor`?  We might also be able to get always with just always using alpha blending for this pass.